script/e2e-test.sh

```diff
@@ -17,28 +17,80 @@
 set -e
 
 ROOT_DIR=`cd "$( dirname "${BASH_SOURCE[0]}" )/.." >/dev/null && pwd`
-IMG_MODIFIER=${1:?}
-DEV_TAG=${2:?}
+DEV_TAG=${1:?}
+IMG_MODIFIER=${2:-""}
+HELM_TLS_FLAGS="--tls --tls-ca-cert ca.cert.pem --tls-cert helm.cert.pem --tls-key helm.key.pem"
```

Comment:
maybe HELM_CLIENT_TLS_FLAGS?

Are you missing `--tls-verify`? Otherwise `tls-ca-cert` is not needed nor used.